recipe/meta.yaml

```diff
@@ -36,19 +37,17 @@ requirements:
     - {{ compiler('c') }}
     - sysroot_linux-64 2.17  # [linux]
   host:
-    - expat   # [linux]
-    - nspr    # [linux]
-    - nss     # [linux]
-    - sqlite  # [linux]
+    - expat {{ expat }}   # [linux]
+    - nspr 4.35           # [linux]
+    - nss 3.89.1          # [linux]
+    - sqlite {{ sqlite }} # [linux]
   run:
     - expat   # [linux]
     - nspr    # [linux]
     - nss     # [linux]
     - sqlite  # [linux]
     - fontconfig         # [linux]
-    - fonts-conda-forge  # [linux]
-    - __osx >=10.11      # [osx]
-    - mathjax =2.7
+    - mathjax 2.7.5
```

Comment:
In logs:
`Adding .* to spec 'mathjax 2.7.5' to ensure satisfiability.  Please consider putting {{ var_name }}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{ pin_compatible() }} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level`